### DISCUSSION TITLE
Birt ROM documentation on new site

### DISCUSSION THREAD

**Discussion by steven-peeters1** [eclipse-birt/birt#844] [Q&A]
Looking for the Birt ROM documentation via Google shows the eclipse.org site at the top result. 
Clicking in this links to the new Birt site that does not (from what I can see) have the ROM Documentation.
I could only find the documentation on an opentext.com site. 
I'm wondering if the Birt ROM documentation will be hosted on the new site?

**Comment by wimjongman** [comment]
I guess this is an important part of the documentation. Can you help?

**Reply by wimjongman** [reply]
The old documentation is here:

https://git.eclipse.org/c/www.eclipse.org/birt.git/tree/phoenix/ref

**Reply by wimjongman** [reply]
@chloetz can you take a look at this?
